File: Modul 5/Bahasa C/PRAK505-2310817120001-DinaIzzatiElfadheya.c
#include <stdio.h>

void Biodata(int tahunLahir, char Nama[], char Asal[]) {
    int tahun_sekarang = 2023;
    int umur = tahun_sekarang - tahunLahir;

    printf("Perkenalkan Nama Saya : %s\n", Nama);
    printf("Umur Saya : %d\n",umur);
    printf("Saya Adalah Angkatan : %d\n", tahun_sekarang);
    printf("Asal Saya dari : %s\n", Asal);
}

int main() {
    int tahunLahir;
    char Namaku[20], Asal[15];

    printf("Input\n");
    scanf("%d", &tahunLahir);
    scanf(" %[^\n]%*c", &Namaku);
    scanf(" %[^\n]%*c", &Asal);
    printf("\nOutput\n");
    Biodata(tahunLahir, Namaku, Asal);
    return 0;
}

File: Modul 5/Bahasa Python/PRAK505-2310817120001-DinaIzzatiElfadheya.py
def Biodata(tahun_lahir, nama, asal):
    tahun_sekarang = 2023
    umur = tahun_sekarang - tahun_lahir

    print(f"Perkenalkan Nama Saya : {nama}")
    print(f"Umur Saya : {umur}")
    print(f"Saya Adalah Angkatan : {tahun_sekarang}")
    print(f"Asal Saya dari : {asal}")

print("Input")
tahun_lahir = int(input())
namaku = input()
asal = input()

print("\nOutput")
Biodata(tahun_lahir, namaku, asal)
